tests/page-objects/collection-search-input.ts

```diff
@@ -29,29 +29,19 @@ export class CollectionSearchInput {
 
   async queryFor(query: string) {
     await this.formInputSearchPage.fill(query);
-    await this.formInputSearchPage.press('Enter');
-  }
-
-  async validateSearchInput(query: string) {
-    expect(await this.formInputSearchPage.inputValue()).toBe(query);
+    await this.btnCollectionSearchInputGo.click();
+    await this.page.waitForTimeout(3000); // Wait for search results to load
```

Comment:
NOTE: It's unfortunate that we need to force this delay in here. Can't we instead wait for detecting some DOM element?

Reply:
updated